### ISSUE TITLE
expandedSidePanelWeight doesn't seem to work

### ISSUE THREAD

**Issue by theherk** [jesseduffield/lazygit#3754]
**Describe the bug**

As far as I can tell, modifying the value of `expandedSidePanelWeight` doesn't seem to alter the behavior of the interface.

**To Reproduce**

1. Set `expandedSidePanelWeight` to anything other than 2.

**Expected behavior**

My understanding is that if I set the value to 3 or 4, the height of the focused side panel will be 3 or 4 (respectively) times greater than that of the unfocused panes.

**Screenshots**

![Screenshot 2024-07-17 at 09 53 08](https://github.com/user-attachments/assets/04f0e3de-af64-4356-92f4-576f7cbf59d4)
![Screenshot 2024-07-17 at 09 53 16](https://github.com/user-attachments/assets/5de26234-6ae3-4ef5-80fe-eee6b4d86d90)
![Screenshot 2024-07-17 at 09 53 26](https://github.com/user-attachments/assets/de1f7de1-e92f-4f82-a29a-73668663e143)

**Version info:**

```
ᐅ lazygit -v
commit=, build date=, build source=homebrew, version=0.43.1, os=darwin, arch=arm64, git version=2.45.2
```

```
ᐅ git --version
git version 2.45.2
```

**Comment by stefanhaller** [comment]
Works for me.

Are you sure that you are using the right location for the lazygit config file? Type `1 e` from within lazygit to make sure you are editing the right one.

**Comment by theherk** [comment]
Yes, as all the other updates are working. However, I set the value to `"4"` just to see if I was mistaken about it being int. It then failed to start with:

```
ᐅ lazygit
2024/07/17 10:05:32 The config at `/Users/h4s/.config/lazygit/config.yml` couldn't be parsed, please inspect it before opening up an issue.
yaml: unmarshal errors:
  line 30: cannot unmarshal !!str `3` into int
```

So I set it back... and now it works. No clue what that was about. Toggling `expandFocusedSidePanel` was working just fine. ¯\_(ツ)_/¯ 

I'll close that, because I don't know that there is an issue to address since I cannot recreate it now, but something fishy was going on. Strange.
